test/support/taxon_helpers.rb

```diff
@@ -0,0 +1,82 @@
+module TaxonHelpers
+  # This taxon has grandchildren
+  def funding_and_finance_for_students(params = {})
+    {
```

Comment:
If you wrap these items in a random example (`merge_and_validate`), they'll be more robust against changes in the schemas.

For example see: https://github.com/alphagov/contacts-frontend/pull/69/commits/885c2733501ccf1060ec2b8334eafdc6e9af95d7#diff-93c10073ef45397d34e244e5d41cc6df